src/command_modules/azure-cli-acr/README.rst

```diff
@@ -72,78 +73,78 @@ Get a container registry
 ::
 
     Command
-        az acr show: Get a container registry.
+        az acr show: Gets the properties of the specified container registry.
 
     Arguments
-        --name -n [Required]: Name of container registry.
+        --name -n [Required]: The name of the container registry.
         --resource-group -g : Name of resource group.
 
 Update a container registry
 -------------
 ::
 
     Command
-        az acr update: Update a container registry.
-    
+        az acr update: Updates a container registry with the specified parameters.
+
     Arguments
-        --name -n      [Required]: Name of container registry.
-        --admin-user-enabled -a  : Whether the admin user account is enabled.  Allowed values: false,
-                                   true.
-        --resource-group -g      : Name of resource group.
-        --storage-account-name -s: Name of an existing storage account.
-        --tags                   : Space separated tags in 'key[=value]' format. Use "" to clear
-                                   existing tags.
-    
+        --name -n   [Required]: The name of the container registry.
+        --admin-enabled       : The value that indicates whether the admin user is enabled.  Allowed
```

Comment:
You could make this more concise. "Indicates whether the admin user is enabled."